Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.6.0
-COPY . .
+FROM python:3.7.3
 RUN pip install flask
 RUN pip install requests
 RUN pip install minio
 RUN pip install wes-service
-CMD [ "python", "./Listener.py"]
+COPY listener.py /srv/server.py
```

Comment:
@SamarthVP take a look here: it is better to run COPY commands last as they will always cause the container to change. If the RUN steps do not change (ie. dependencies do not change/update), Docker can used a cached version of the container image to rebuild the container, saving us time. By moving COPY near the end, we avoid having to re-run these RUN steps!

Reply:
Oh yeah, that's smart